openedx/features/ucsd_features/additional_registration_fields.py

```diff
@@ -13,12 +13,27 @@ class AdditionalRegistrationFieldsForm(ModelForm):
     """
     The fields on this form are derived from the AdditionalRegistrationFields model in models.py.
     """
-    def __init__(self, *args, **kwargs):
-        super(AdditionalRegistrationFieldsForm, self).__init__(*args, **kwargs)
 
     class Meta(object):
         model = AdditionalRegistrationFields
         fields = ('gender_nb', 'ethnicity', 'age', 'education', 'howheard')
+        error_messages = {
```

Comment:
You can also use help text with error messages here like given below
```
class Meta(object):
        error_messages = {
             ....
        }
        help_texts = {
            'registration_number': 'Enter your registration number provided from your school.',
            'first_name': 'Enter your first name.',
            'surname': 'Enter your surname.',
            'other_name' : 'Enter your nick name.',
            'current_level': 'Select your current level in school during admission.',
            'mobile_number': 'Enter your mobile number.',
            'department': 'Select your department',
            'programme': 'Select the programme that you have been enrolled in.',
        }

```

Reply:
I don't think these are required as all of the custom fields are dropdowns. So there are limited, self-descriptive options there. Also, the message will be hidden behind the dropdown regardless.
![image](https://user-images.githubusercontent.com/42166091/79427747-98b9c480-7fde-11ea-88b8-0fce03a94956.png)

Reply:
![image](https://user-images.githubusercontent.com/42294172/79428093-17aefd00-7fdf-11ea-80c1-8b9a3283695d.png)
No, the text didn't hide behind the dropdown. When you click on the drop-down menu for the first time it shows the help text

Reply:
I've added the help texts/